backend/tet/events/api/v1/permissions.py

```diff
@@ -1,12 +1,41 @@
+from django.conf import settings
+from django.http import HttpRequest
 from rest_framework.permissions import BasePermission
 
 
-class TetAdminPermission(BasePermission):
+class TetAPIPermission(BasePermission):
     """
-    Permission check for managing city TET postings.
+    Permission to access the Linked Events API as authorized.
+
+    There are three kinds of possible logins:
+
+      1. Users logged in via AD belonging to a group listed in `settings.ADFS_CONTROLLER_GROUP_UUIDS`.
+         This group should contain all City of Helsinki users needing to use this servicing.
+         These users are logged in as staff users and should be allowed access to their own
+         TET postings.
+      2. Users logged in via suomi.fi (puolesta asiointi)
+         The service works if we can find a business_id for the user and the user is allowed access to
+         the postings of their company.
+      3. Users logged in via AD NOT belonging to a group listed in `settings.ADFS_CONTROLLER_GROUP_UUIDS`.
+         This could be anyone with a Microsoft account. The ADFS setup might prevent these logins,
+         but we cannot be sure about this, so we need to check that not anyone is allowed to creating
+         a TET posting as an employee of the city of Helsinki.
```

Comment:
"so we need to check that not anyone is allowed to creating"

Do you mean this?
"so we need to check that they are not allowed to create"